src/ui/components/Icon/styles.scss

```diff
@@ -37,3 +38,26 @@
   background: url('./user-dark.svg') center no-repeat;
   background-size: contain;
 }
+
+.Icon-browse {
+  background: url('./bullet-list.svg') center no-repeat;
+  background-size: cover;
+  height: 20px;
+  width: 20px;
+}
+
+.Icon-themes {
+  background: url('./themes.svg') center no-repeat;
+  background-size: cover;
+  height: 50px;
+  min-width: 41.5px;
```

Comment:
These values (and the ones below) seem very strange. Should the SVG have better sizes or defaults?